weather.css

```diff
@@ -0,0 +1,54 @@
+
+#font
+ {
+    color: white;
+    font-size: 100px;
+    font-family:system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  }
+  table{
+    border-collapse: collapse;
+    padding: 20px;
+    color: white;
+    font-size: 65px;
+    font-family:system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+
+  }
+  
+
+  #slika{
+    width: 100px;
+    height: 100px;
+  }
+  
+  .search input[type=text] {
+    padding: 6px;
+    border: none;
+    margin-top: 8px;
+    margin-right: 16px;
+    font-size: 17px;
+  }
+
+  .button {
+    background-color:dodgerblue; 
+    border: none;
+    color: white;
+    padding: 15px 32px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+  }
+
+  .button:hover{
+    opacity: 50%;
+  }
+
+  #demo, h2{
+    color: white;
+    font-size: 50px;
+    font-family:system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  }
+
+  body{
```

Comment:
<img width="1918" alt="Screenshot_1" src="https://github.com/Wolitz/WeatherApp/assets/85948457/ba4c2e59-62a9-4b4d-bdf2-b651619e692d">
I am using 4k display so background looks broken, on lower resolutions it looks ok. To solve this you can add:
` min-height: 100vh;
  overflow: hidden;`
  this is not optimal solution because we are hiding scrollbar but in this case it's fine

Reply:
You can add those lines below background-image

Reply:
Added the lines but now "WS" is not showing on my Mac screen , I will try to find an optimal solution but for now it works.